### ISSUE TITLE
Broken Twitter avatars, verbose Twitter links

### ISSUE THREAD

**Issue by aardrian** [Devographics/Monorepo#166]
In Firefox using "Strict" for my privacy mode, Twitter avatars are missing (this image is from the home page):
<img width="556" alt="Special Thanks and then four names in a 2×2 grid; each name is preceded by the same name in an invisible circle which clips some of the letters." src="https://user-images.githubusercontent.com/1376607/206200745-da65cb64-85f3-4782-93d8-48ceb6336758.png">

Thrilled that there is alt text, but the missing images also highlight that the links are duplicated. This adds unnecessary tab-stops and redundancy, not to mention it might make users think the different links go to different places. I suggest removing the links from the avatars and giving them `alt=""`.

As I tab through the page in NVDA, I get this (`Tab` is me pressing the Tab key):

> Lea Verou link  
> `Tab`
> Lea Verou link heading level 4
> `Tab`
> Nicole Sullivan visited link  
> `Tab`
> Nicole Sullivan visited link heading level 4
> `Tab`
> Philip Jägenstedt link  
> `Tab`
> Philip Jägenstedt link heading level 4
> `Tab`
> Christopher Kirk-Nielsen link  
> `Tab`
> Christopher Kirk-Nielsen link heading level 4

If I just let the page read through, from the "Special Thanks" heading I get:

> Special Thanks  heading    level 3
> link    Lea Verou
> heading    level 4  link    Lea Verou
> Survey Design
> visited  link    Nicole Sullivan
> heading    level 4  visited  link    Nicole Sullivan
> Survey Design
> link    Philip Jägenstedt
> heading    level 4  link    Philip Jägenstedt
> Survey Design
> link    Christopher Kirk-Nielsen
> heading    level 4  link    Christopher Kirk-Nielsen
> Logo Design

Related, I do not think those links need to _also_ be a heading (at any level).

**Comment by SachaG** [comment]
Thanks, I'll have to look into this.

**Comment by eric-burel** [comment]
@SachaG what's the state of this ? We already hosts the image ourselves ?

**Comment by SachaG** [comment]
Yes, all avatars are available at https://assets.devographics.com/avatars/${entity.id}.jpg, for example https://assets.devographics.com/avatars/sacha_greif.jpg

They are currently copied from Twitter but I'm not sure our Twitter API integration still works, so that may be something to look into next time I add new people and need to load their avatars.
